### ISSUE TITLE
Add line & donut chart

### ISSUE THREAD

**Issue by timwis** [timwis/vizwit#94]
Really just a matter of changing one config line in the bar & pie charts, respectively

**Comment by marks** [comment]
@timwis: I've been looking at how to do this but am not seeing how to use an additional card config key-value pair (like `chartSubtype`) to inform/change the chart settings. Suggestions on how to achieve this?

* **Line:** for this, I think we just need to remove the `fillAlpha` from the graph settings
* **Pie:** I believe the "pie" is actually already a donut since there is an inner radius (so let's have a config key that lets the radius to be passed in (and if there is none, it will be a pie)

**Comment by timwis** [comment]
yes @marks I agree -- I think I was originally thinking that a user configuring the json file wouldn't need to know that "bar" and "line" are based on the same code; they'd just specify which one they want as the `chartType` rather than subtype
